localstack-core/localstack/services/events/provider.py

```diff
@@ -199,6 +199,8 @@ def validate_event(event: PutEventsRequestEntry) -> None | PutEventsResultEntry:
             "ErrorCode": "InvalidArgument",
             "ErrorMessage": "Parameter Detail is not valid. Reason: Detail is a required argument.",
         }
+    elif event.get("Detail") and len(event["Detail"]) >= 262144:
```

Comment:
We can assume that the detail exists here of it would have gone through line 197
```suggestion
    elif len(event["Detail"]) >= 262144:
```